spacy/matcher.pyx

```diff
@@ -546,16 +546,18 @@ cdef class PhraseMatcher:
     cdef Matcher matcher
     cdef PreshMap phrase_ids
     cdef int max_length
+    cdef attr_id_t attr
     cdef public object _callbacks
     cdef public object _patterns
 
-    def __init__(self, Vocab vocab, max_length=0):
+    def __init__(self, Vocab vocab, max_length=0, attr='ORTH'):
         if max_length != 0:
             deprecation_warning(Warnings.W010)
         self.mem = Pool()
         self.max_length = max_length
         self.vocab = vocab
         self.matcher = Matcher(self.vocab)
+        self.attr = self.vocab.strings[attr]
```

Comment:
If the user passes in e.g. `ORTH` as an ID, this will cause an error. So we probably want something like

```
if isinstance(attr, long):
    self.attr = attr
else:
    self.attr = self.vocab.strings[attr]
```